rowview.php

```diff
@@ -172,6 +174,11 @@
 	}
 ?>
 </div>  <!-- END div.main -->
+<?php
+	if ($person->SiteAdmin || $person->RackAdmin) {
```

Comment:
This will prevent anyone from moving equipment in the rowview unless they are a rack/site admin.  This could lead to confusing behavior if someone has the correct rights to move equipment in the cabinet view they would expect to be able to do it here too.

Reply:
Indeed, added `$person->WriteAccess` to that too. Thanks!